src/app/contactmanager/components/sidenav/sidenav.component.html

```diff
@@ -7,7 +7,7 @@
       Contacts
     </mat-toolbar>
     <mat-nav-list>
-        <mat-list-item *ngFor="let user of users | async" [routerLink]="['/contactmanager', user.id]">
+        <mat-list-item *ngFor="let user of userService.users()" [routerLink]="['/contactmanager', user.id]">
```

Comment:
Another thing you may want to consider is updating the structural directives in your templates to the new control flow. The Angular team is heavily pushing the new control flow. Our team at Cisco has already adopted it even though it is still technically in dev preview.

https://angular.io/guide/control_flow

Reply:
Great advice again, replaced all ngFor, ngIf directives!